### DISCUSSION TITLE
Does PrimeReact DataTable Support Drag-and-Drop Row Sorting?

### DISCUSSION THREAD

**Discussion by v-adbarve** [primefaces/community#2646] [PrimeReact] [answered]
Hi everyone,
I'm currently working on a project using PrimeReact and I'm utilizing the DataTable component to display a list of items. I need to implement drag-and-drop functionality to allow users to sort rows within the table.
My Questions:
1.	Does PrimeReact's DataTable component natively support drag-and-drop functionality for sorting rows?
2.	If not, is there a recommended workaround  with PrimeReact to achieve this functionality?

**Comment by melloware** [comment, marked as answer]
Isn't that this example: https://primereact.org/datatable/#reorder
